packages/testing/scripts/download-test-server.mjs

```diff
@@ -2,10 +2,30 @@ import stream from 'node:stream';
 import util from 'node:util';
```

Comment:
tested manually?

Reply:
I tested manually but this path is exercised in ci too